src/renderers/dom/shared/ReactDOMComponent.js

```diff
@@ -322,6 +322,70 @@ function trapBubbledEventsLocal() {
         ),
       ];
       break;
+    case 'video':
+      inst._wrapperState.listeners = [
```

Comment:
My understanding was that this function was for events that can't be caught at the global level and need to be attached to the actual dom node. For example, media events don't bubble and can only be received on the media node.